pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -125,17 +125,19 @@ type CAOptions struct {
 
 // RunCA will start the cert signing GRPC service on an existing server.
 func RunCA(grpc *grpc.Server, cs kubernetes.Interface, opts *CAOptions) {
```

Comment:
Where is the flag for this module?

Reply:
Currently it's controlled by having the JWT mounted by installer, in /var/run/secrets/... 
I can add a more explicit config.

Reply:
But not sure it's ideal - if the file can't be created because k8s doesn't have this feature, the flag will force a crash or
bad config. It's quite hard for installer to guess if k8s has the token feature available - it is an optional mount, and 
pilot cand determine if the required tokens are supported or not. Without the tokens - we can't use SDS.

Reply:
Added comments.